client/components/theme/style.scss

```diff
@@ -158,6 +160,16 @@ $theme-info-height: 54px;
 	height: $theme-info-height;
 }
 
+.theme__installing {
+	background: transparentize($gray-dark, 0.5);
+	width: 100%;
+	left: 0;
+	top: 0;
+	bottom: 54px;
```

Comment:
54px comes from Theme component CSS. It is just how much space the lower part takes (the one with ellipsis menu) I am changing it to already used `$theme-info-height` variable that holds that value for SSOT. And the value itself is not scaled. Name + ellipsis have always the same height.